test/integration/scripts/create-email-provider.sh

```diff
@@ -0,0 +1,4 @@
+EXIT_STATUS=0
+auth0 api POST "emails/provider" --data '{"name":"mandrill","credentials":{"api_key":"some-api-key"}}' || EXIT_STATUS=$?
+auth0 api PATCH "emails/provider" --data '{"enabled":true}' || EXIT_STATUS=$?
+exit $EXIT_STATUS
```

Comment:
Does bash set `-e` by default?

Reply:
@Widcket did a bit of digging and you're right 👍🏻, on an ubuntu docker image it's not set by default. It was on my machine just because I have shell options set globally. 

However there's still a problem with this script even if we add `set -e`, if we use the api command and we get back a 400 or 500 the exit code will still be 0.